doc/delta-kernel-book/README.md

```diff
@@ -0,0 +1,19 @@
+# delta kernel mdbook
+
+This book serves as a user guide + developer guide + overall docs for the kernel. In addition to our
+docs.rs and top-level README, the three provide comprehensive documentation for the project.
+
+## prerequisites
+This book is built with [`mdbook`]. Install the latest version with `cargo install mdbook` or whatever
+their latest docs say.
+
+
+## building
+The book is built in CI and deployed to github pages. TODO link to the page here. When working on
+the book locally you can preview changes with mdbook local server:
```

Comment:
There currently is a project under way to update the delta.io site.

This includes also the docs sites for other delta projects. As we are just getting stated I think we should aim to integrate there right away?

Reply:
oh nice - do we have a spot for the rust kernel? where are we envisioning this should go?